docs/public-api-changes-NIO1-to-NIO2.md

```diff
@@ -86,3 +86,5 @@
 - change  `ChannelPipeline.addHandler(_:after:)` to  `ChannelPipeline.addHandler(_:postion:)` where `position` can be `.first`, `.last`, `.before(ChannelHandler)`, and `.after(ChannelHandler)`
 - Change `HTTPServerProtocolUpgrader` `protocol` to require `buildUpgradeResponse` to take a `channel` and return an `EventLoopFuture<HTTPHeaders>`.
 - `EmbeddedChannel.writeInbound/Outbound` are now `throwing`
+- `HTTPMethod.hasRequestBody` and the `HTTPMethod.HasBody` type have been removed from the public API
+- `CircularBuffer.Index` is no longer `Strideable`
```

Comment:
This dos not seem to belong to this PR ?

Reply:
@normanmaurer yeah, I sneaked that one in ;)